src/plugins/ini/ini.c

```diff
@@ -23,17 +26,20 @@ int elektraIniClose(Plugin *handle, Key *parentKey);
 
 #include "contract.h"
 
+#define INTERNAL_ROOT_SECTION "GLOBALROOT"
```

Comment:
Any suggestions for a better name ? 
The name should be something easy to remember, because it can be used to create global ini keys with kdb set having the same name as a section key. e.g. having `[section1]` and `section1 = i_am_a_global_key`